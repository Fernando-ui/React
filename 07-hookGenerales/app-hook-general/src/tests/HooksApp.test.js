import { shallow } from 'enzyme';
import { HooksApp } from '../HooksApp';
import '@testing-library/jest-dom'

describe('Pruebas con componentesm Hooks', () => {
  
  let wrapper = shallow (<HooksApp/>);


  test('Debe de regresar el valor del DOM', () => {


    expect(wrapper).toMatchSnapshot();


  });
  
});
