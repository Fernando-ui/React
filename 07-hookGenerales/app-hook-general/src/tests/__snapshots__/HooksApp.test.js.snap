// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas con componentesm Hooks Debe de regresar el valor del DOM 1`] = `
<Fragment>
  <h1>
    Hola mundo
  </h1>
</Fragment>
`;
